tools/deploy_ML_translated_documentation.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# Script for publishing translated ML documentation to staging site.
+
+# Non-travis variables used by this script.
+ML_SOURCE_REPOSITORY="https://github.com/Qiskit/qiskit-machine-learning.git"
+SOURCE_DOC_DIR="docs/_build/html"
+SOURCE_DIR=`pwd`
+
+STABLE_VERSION=`cat ./qiskit_machine_learning/VERSION.txt`
+FORMATED_VERSION=`echo $STABLE_VERSION | cut -d "." -f -2`
+
+curl https://downloads.rclone.org/rclone-current-linux-amd64.deb -o rclone.deb
+sudo apt-get install -y ./rclone.deb
+
+RCLONE_CONFIG_PATH=$(rclone config file | tail -1)
+
+set -e
+
+# Clone the sources files and po files to ml_docs_source/
+git clone $ML_SOURCE_REPOSITORY ml_docs_source
+git checkout stable/$FORMATED_VERSION
```

Comment:
Don't you need to change directories to `ml_docs_source` before this will work?

Reply:
Fixed in https://github.com/qiskit-community/qiskit-translations/pull/686/commits/1371814965d55aa79a22ad8b89c690b920a933a1

Reply:
I think that commit actually introduces a separate bug below on L38 where you copy the contents of `ml_docs_source/docs` but you're in the `ml_docs_source` dir already

Reply:
Fixed in https://github.com/qiskit-community/qiskit-translations/pull/686/commits/a59c6f44294978d8798903cce0d41a35ac16864e